website/docs/components/code-block/partials/guidelines/guidelines.md

```diff
@@ -69,6 +69,24 @@ Line numbers are displayed by default and can make longer blocks of code and sni
 In the Figma component, the code examples have the appropriate number of lines by default but must be manually hidden or shown to match the length of custom snippets.
 !!!
 
+## Height toggle button
+
+When displaying longer code snippets, it can be helpful to let users expand and collapse the content. Setting a `maxHeight` on the Code Block limits how much is shown by default. If the content exceeds this height, a **Show more code** button appears at the bottom of the block, allowing users to expand it. Clicking again collapses it back.
```

Comment:
In some of my other suggestions I noted using "interacting" vs "clicking" which is more inclusive, so noting that here.

```suggestion
When displaying longer code snippets, it can be helpful to let users expand and collapse the content. Setting a `maxHeight` on the Code Block limits how much is shown by default. If the content exceeds this height, a **Show more code** button appears at the bottom of the block, allowing users to expand it. Interacting with the button again collapses it to it's original height.
```